src/python/WMCore/WMRuntime/Monitors/PerformanceMonitor.py

```diff
@@ -188,29 +184,22 @@ def periodicUpdate(self):
 
         # Now we run the monitor command and collate the data
         cmd = self.monitorBase % (stepPID, stepPID)
-        stdout, stderr, retcode = subprocessAlgos.runCommand(cmd)
+        stdout, _stderr, _retcode = subprocessAlgos.runCommand(cmd)
 
         output = stdout.split()
-        if not len(output) > 7:
+        if not len(output) > 6:
             # Then something went wrong in getting the ps data
             msg = "Error when grabbing output from process ps\n"
             msg += "output = %s\n" % output
             msg += "command = %s\n" % self.monitorCommand
             logging.error(msg)
             return
-        # FIXME: making it backwards compatible. Keep only the "else" block in HG1801
-        if self.maxRSS is not None and self.maxRSS >= (1024 * 1024):
-            # then workload value is still in KiB (old way)
-            rss = int(output[2])
-            vsize = int(output[3])
-        else:
-            # ps returns data in kiloBytes, let's make it megaBytes
-            # I'm so confused with these megabytes and mebibytes...
-            rss = int(output[2]) // 1000  # convert it to MiB
-            vsize = int(output[3]) // 1000  # convert it to MiB
+
+        # ps returns data in kiloBytes, let's make it megaBytes
+        # I'm so confused with these megabytes and mebibytes...
+        rss = int(output[2]) // 1000  # convert it to MiB
```

Comment:
This is converting to MegaBytes (MB), not MebiBytes (MiB) as indicated in the comment above, so the conversion comment in this line is wrong.

Reply:
Thanks Kenyi! Noted it in my brain for the next time I touch this module.